github_releaser/cmd/push.py

```diff
@@ -0,0 +1,25 @@
+import click
+import os
+
+from github_releaser import GithubReleaser
+
+
+@click.command(name="push-files", help="Push new files to a branch")
+@click.option("--account", "--a", required=True, help="Account")
+@click.option("--repository", "--r", required=True, help="Repository")
+@click.option("--token", help="GitHub's API token")
+@click.option("--branch-name", "--b", required=True, help="The branch name. Default: master")
```

Comment:
I think we can add a default="master" here, remove the requirement and in the push function we don't need to check if the branch have been specified or not. I think I did the same for the other commands, it's good to change as well.

Reply:
I left the check for master branch just to not add check if to provide params or not in the requests